dotnet-desktop-guide/framework/wpf/advanced/how-to-use-application-resources.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Use Application Resources"
+description: Learn how to use Application Resources sfo that resources defined at the application level can be accessed by all other pages that are part of the application.
```

Comment:
```suggestion
description: Learn how to use Application Resources so that resources defined at the application level can be accessed by all other pages that are part of the application.
```